pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -661,27 +783,33 @@ def install_packages(self, callback, timeout=20):
             log.debug("The transaction process exited with %s.", process.exitcode)
 
     @staticmethod
-    def _run_transaction(base, display):
+    def _run_transaction(base, transaction, progress):
```

Comment:
Do you need to pass transaction?  Can't you just access the class _transaction variable from within the method?

Reply:
Probably not. I will take a look at it, but this will likely affect tests, so it's not as easy as to just remove it.

Reply:
So, I just added the parameter description to the docstring.